app/services/CalculationService.scala

```diff
@@ -41,10 +41,10 @@ class FinancialDataService @Inject()(val lastTaxCalculationConnector: LastTaxCal
     } yield (lastCalc, calcBreakdown) match {
       case (calc: LastTaxCalculation, breakdown: CalculationDataModel) =>
         Logger.debug("[FinancialDataService] Retrieved all Financial Data")
-        CalcDisplayModel(calc.calcTimestamp, calc.calcAmount, Some(breakdown))
+        CalcDisplayModel(calc.calcTimestamp, calc.calcAmount, Some(breakdown), calc.crystalisedFlag.getOrElse("Uncrystalised"))
```

Comment:
Here we'd use the `Crystalised` caseobject of the sealedtrait or enumeration value instead of the string once the model is changed